lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -272,16 +272,12 @@ function setSubscriptionsAndTimeouts(featureInfoSection, feature) {
         setCurrentFeatureValues(changedFeature, featureInfoSection.props.clock);
     });
     if (featureInfoSection.isFeatureTimeVarying(feature)) {
-        if (defined(featureInfoSection.props.clock) && defined(featureInfoSection.props.clock.onTick)) {
+        if (defined(featureInfoSection.props.clock) && defined(featureInfoSection.props.clock.definitionChanged)) {
             featureInfoSection.setState({
-                removeClockSubscription: featureInfoSection.props.clock.onTick.addEventListener(function(clock) {
+                removeClockSubscription: featureInfoSection.props.clock.definitionChanged.addEventListener(function(clock) {
```

Comment:
Is this subscription needed anymore? I think you might just be able to change the UI to access `catalogItem.currentTime` and the observability will make the UI automatically refresh when it changes.